step-functions-plugins/step-functions-plugins-java/step-functions-plugins-java-def/src/main/java/step/plugins/java/GeneralScriptFunctionType.java

```diff
@@ -49,4 +52,16 @@ public GeneralScriptFunction newFunction() {
 		function.getScriptLanguage().setValue("java");
 		return function;
 	}
+
+	@Override
+	public Map<String, String> getHandlerProperties(GeneralScriptFunction function) {
+		String language = getScriptLanguage(function);
+		if (language != null) {
+			String errorHandler = configuration.getProperty("script." + language + "_errorhandler", null);

```

Comment:
@cl-exense I would recommend to use the same namespace as the other configuration items of the AbstractScriptFunctionType: for instance: plugins.<language>.errorhandler

Reply:
OK, thanks -- done, now using plugins.`${language}`.errorhandler